src/components/layout/nav/UserMenu.tsx

```diff
@@ -23,18 +24,21 @@ import { useIsMobile } from '@/hooks/useIsMobile';
 import useAuth from '@/modules/auth/hooks/useAuth';
 import { useHmisAppSettings } from '@/modules/hmisAppSettings/useHmisAppSettings';
 
-const UserMenu: React.FC = () => {
+interface UserMenuProps {
+  hideName?: boolean;
+}
+const UserMenu: React.FC<UserMenuProps> = ({ hideName }) => {
   const popupState = usePopupState({ variant: 'popover', popupId: 'userMenu' });
   const { user, logoutUser } = useAuth();
   const { manageAccountUrl, warehouseUrl, warehouseName } =
     useHmisAppSettings();
-  const isMobile = useIsMobile();
+  const isTiny = useIsMobile('sm');
```

Comment:
This leads to the desktop-style buttons getting displayed on `md` size screens, which I think is not intentional, right?

<img width="338" alt="Screenshot 2024-08-13 at 10 53 43 AM" src="https://github.com/user-attachments/assets/91cf90ec-49ca-40be-b023-57d420880a81">

Reply:
I ended up rolling back the change because there were a lot of downstream effects of treating `sm` as mobile here but `md` as mobile elsewhere. The MUI default for `md` is 900px which is quite large, even larger than the norm for tablet size which appears to be 768px.

So, to simplify this, I rolled back the change so that we are still using `md`, but I reduced the `md` breakpoint to 769 in the theme. This negatively affects the Enrollment Dashboard when viewed in the 800-900px width page _with the nav menu open_, which should be fixed, but I think is an OK tradeoff. (The benefit is that desktop users on 800-900 are not forced to use the mobile navigation menu).

Thoughts on that @martha @ttoomey?

Reply:
Oops, the issue @martha pointed out is still occurring, hold on.